### DISCUSSION TITLE
wiki: broken link to alarmsystem how-to ?

### DISCUSSION THREAD

**Discussion by kamaradski** [OpenCCU/OpenCCU#1141] [🇺🇸 Q&A] [answered]
There used to be a page but now it's gone?

**not here:** https://github.com/jens-maus/RaspberryMatic/wiki/How-To-setup-an-alarm-system
**and not here:** https://github.com/jens-maus/RaspberryMatic/wiki/en.Tipps-AlarmSystem

Could someone with access to the underlying git repo have a look and re-link it ?
With the new layout of the documentation I think it should be linked here now? https://github.com/jens-maus/RaspberryMatic/wiki/en.Tipps-AlarmSystem

**Comment by jens-maus** [comment, marked as answer]
> There used to be a page but now it's gone?
> 
> **not here:** https://github.com/jens-maus/RaspberryMatic/wiki/How-To-setup-an-alarm-system
> **and not here:** https://github.com/jens-maus/RaspberryMatic/wiki/en.Tipps-AlarmSystem
> 
> Could someone with access to the underlying git repo have a look and re-link it ?

The page which you are searching for is here:
https://github.com/jens-maus/RaspberryMatic/wiki/Tipps-AlarmSystem

> With the new layout of the documentation I think it should be linked here now? https://github.com/jens-maus/RaspberryMatic/wiki/en.Tipps-AlarmSystem

Well, better would be if you generate that page yourself (via Copy&Paste) but then please take the time to translate it to german as well and put the german content into https://github.com/jens-maus/RaspberryMatic/wiki/Tipps-AlarmSystem instead

**Reply by kamaradski** [reply]
> Well, better would be if you generate that page yourself (via Copy&Paste) 

Done, I have created "en.Tipps-AlarmSystem"

> please take the time to translate it to german as well

I have left "Tipps-AlarmSystem" in place with a note on top that this page needs translation. Though I can read and fluently speak German, you really don't want me to write anything in German on this wiki :P It would be a mess, hahaha.
I can ask some of my friends if they feel like translating it, but no guarantee they can/will commit to this task.
